src/main/resources/db/changelog/db.changelog-master.yaml

```
@@ -0,0 +1,22 @@
+databaseChangeLog:
+  - changeSet:
+      id: enable-pgcrypto
+      author: Klim Birukou <manulc0rparation@gmail.com>
+      changes:
+        - sql:
+            sql: CREATE EXTENSION IF NOT EXISTS "pgcrypto";
+
+  - changeSet:
+      id: create_user_table
+      author: Klim Birukou <manulc0rparation@gmail.com>
+      changes:
+        - createTable:
+            tableName: user
```

Same for table name
Let's name it as
```suggestion
            tableName: dl_user
```